src/com/company/Main.java

```diff
@@ -230,6 +237,45 @@ public static void main(String[] args) {
 
   }
 
+  private static void addPhoneNumbers (PhoneNumber phoneNumber){
```

Comment:
at the end of this method, you should add the phone number to the "phoneNumbers" field of the Customer class to save it.

Reply:
Not sure what I should do here.

Reply:
@mtelyuk  can you please contact with @mahir0837 for getting help from him to finish the task.

Reply:
ArrayList<PhoneNumber> phoneNumbers = new ArrayList<PhoneNumber>();

    phoneNumbers.add(new PhoneNumber(customerPhoneNumber));

Am I heading in the right direction with this? 
I want to try and do it myself.